search/reverse_geocoder.cpp

```diff
@@ -241,11 +254,12 @@ void ReverseGeocoder::GetNearbyAddress(m2::PointD const & center, double maxDist
 
 bool ReverseGeocoder::GetExactAddress(FeatureType & ft, Address & addr) const
 {
-  if (ft.GetHouseNumber().empty())
+  std::string hn = GetHouseNumber(ft);
```

Comment:
И тогда можно будет так делать:
```suggestion
  std::string const & hn = GetHouseNumber(ft);
```

Reply:
Если это при поиске происходит, то лучше избегать ненужных аллокаций. Мало ли будем выдавать результаты сотнями в списке.